cloud/amazon/sqs_queue.py

```diff
@@ -113,16 +148,23 @@ def create_or_update_sqs_queue(connection, module):
     try:
         queue = connection.get_queue(queue_name)
         if queue:
-            # Update existing
+            # Update existing            
             result['changed'] = update_sqs_queue(queue, check_mode=module.check_mode, **queue_attributes)
-
         else:
             # Create new
             if not module.check_mode:
                 queue = connection.create_queue(queue_name)
                 update_sqs_queue(queue, **queue_attributes)
             result['changed'] = True
-
+        
+        if not module.check_mode:
```

Comment:
Hi, when I'm creating queue, although the queue is successfully created, I don't get the `queue_arn` from ansible. Can I force this attribute to always return `queue_arn`?